en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/security/transport-security.md

```diff
@@ -1,26 +1,92 @@
 |------------|----------------------------------|--------------|
-|protocolName| SSL Protocol to be used| "TLS"|
-|protocolVersions | SSL/TLS protocols to be enabled |"TLSv1.2,TLSv1.1"|
-|ciphers |List of ciphers to be used|"TLS\_ECDHE\_RSA\_WITH\_AES\_128\_CBC\_SHA,TLS\_ECDHE\_ECDSA\_WITH\_AES\_128\_CBC\_SHA256, TLS\_ECDHE\_RSA\_WITH\_AES\_128\_CBC\_SHA256,TLS\_RSA\_WITH\_AES\_128\_CBC\_SHA256,TLS\_ECDH\_ECDSA\_WITH\_AES\_128\_CBC\_SHA256, TLS\_ECDH\_RSA\_WITH\_AES\_128\_CBC\_SHA256,TLS\_DHE\_RSA\_WITH\_AES\_128\_CBC\_SHA256,TLS\_DHE\_DSS\_WITH\_AES\_128\_CBC\_SHA256, TLS\_ECDHE\_ECDSA\_WITH\_AES\_128\_CBC\_SHA,TLS\_ECDHE\_RSA\_WITH\_AES\_128\_CBC\_SHA,TLS\_RSA\_WITH\_AES\_128\_CBC\_SHA, TLS\_ECDH\_ECDSA\_WITH\_AES\_128\_CBC\_SHA,TLS\_ECDH\_RSA\_WITH\_AES\_128\_CBC\_SHA,TLS\_DHE\_RSA\_WITH\_AES\_128\_CBC\_SHA, TLS\_DHE\_DSS\_WITH\_AES\_128\_CBC\_SHA,TLS\_ECDHE\_ECDSA\_WITH\_AES\_128\_GCM\_SHA256,TLS\_ECDHE\_RSA\_WITH\_AES\_128\_GCM\_SHA256 ,TLS\_RSA\_WITH\_AES\_128\_GCM\_SHA256,TLS\_ECDH\_ECDSA\_WITH\_AES\_128\_GCM\_SHA256,TLS\_ECDH\_RSA\_WITH\_AES\_128\_GCM\_SHA256, TLS\_DHE\_RSA\_WITH\_AES\_128\_GCM\_SHA256,TLS\_DHE\_RSA\_WITH\_AES\_128\_GCM\_SHA256,TLS\_DHE\_DSS\_WITH\_AES\_128\_GCM\_SHA256 ,TLS\_ECDHE\_ECDSA\_WITH\_3DES\_EDE\_CBC\_SHA,TLS\_ECDHE\_RSA\_WITH\_3DES\_EDE\_CBC\_SHA,SSL\_RSA\_WITH\_3DES\_EDE\_CBC\_SHA, TLS\_ECDH\_ECDSA\_WITH\_3DES\_EDE\_CBC\_SHA,TLS\_ECDH\_RSA\_WITH\_3DES\_EDE\_CBC\_SHA,SSL\_DHE\_RSA\_WITH\_3DES\_EDE\_CBC\_SHA, SSL\_DHE\_DSS\_WITH\_3DES\_EDE\_CBC\_SHA,TLS\_EMPTY\_RENEGOTIATION\_INFO\_SCSV"|
-|sslVerifyClient|The type of client certificate verification. ("require" or "optional")| "optional"|
+|minimumProtocolVersion|The supported minimum tls version|`TLS1_1`|
+|maximumProtocolVersion|The supported maximum tls version|`TLS1_2`|
+|ciphers|List of ciphers to be used|`ECDHE-ECDSA-AES128-GCM-SHA256, ECDHE-RSA-AES128-GCM-SHA256, ECDHE-ECDSA-AES128-SHA, ECDHE-RSA-AES128-SHA, AES128-GCM-SHA256, AES128-SHA, ECDHE-ECDSA-AES256-GCM-SHA384, ECDHE-RSA-AES256-GCM-SHA384, ECDHE-ECDSA-AES256-SHA, ECDHE-RSA-AES256-SHA, AES256-GCM-SHA384, AES256-SHA`|
+|trustedCertPath|The path where the trusted certificates are stored|`/etc/ssl/certs/ca-certificates.crt`|
+|verifyHostName|Verify the backend hostname with the certificate SAN (Subject Alternative Name)|`true`|
+|disableSslVerification|Disable SSL verification with backend clusters|`false`|
+
+### Add a Certificate to Choreo Connect Router as a Trusted Certificate
+Choreo Connect supports certificates in `pem` format. The trusted certificates are added as a single file which contains all the certificates.
+To add a new certificate please follow the steps below.
+
+!!!Important
+    Mount the default certificate location of the Choreo Connect router in `<CHOREO-CONNECT_HOME>/docker-compose/choreo-connect or choreo-connect-with-apim/docker-compose.yaml` file. This will allow modifying the certificates easily without loggin to the container.
```

Comment:
```suggestion
    Mount the default certificate location of the Choreo Connect router in `<CHOREO-CONNECT_HOME>/docker-compose/choreo-connect or choreo-connect-with-apim/docker-compose.yaml` file. This will allow modifying the certificates easily without logging in to the container.
```